src/generalChecks/index.js

```diff
@@ -108,4 +108,28 @@ module.exports = testPath => {
       });
     });
   });
+
+  read(`${testPath}/Jenkinsfile`, 'utf8', err => {
+    if (err) {
+      console.log(red, 'No existe un Jenkinsfile');
+      return;
+    }
+    console.error(green, 'Existe un Jenkinsfile');
+  });
+
+  read(`${testPath}/.woloxci/config.yml`, 'utf8', err => {
+    if (err) {
+      console.log(red, 'No existe un config.yml en .woloxci');
+      return;
+    }
+    console.error(green, 'Existe un config.yml en .woloxci');
+  });
+
+  read(`${testPath}/.woloxci/Dockerfile`, 'utf8', err => {
+    if (err) {
+      console.log(red, 'No existe un Dockerfile');
+      return;
+    }
+    console.error(green, 'Existe un Dockerfile');
+  });
```

Comment:
Can we check a little bit more of each file? Now we are checking only if it exist. But you can have a file called
`Dockerfile` with nothing inside and the check would be green. Maybe we can upgrade this to make a more in depth check. What do you think?